models/stg_iterable.yml

```diff
@@ -0,0 +1,419 @@
+version: 2
+
+models:
+  - name: stg_iterable__campaign_history
+    description: Table containing a history of campaigns.
+    loaded_at_field: _fivetran_synced
```

Comment:
i think `loaded_at_field` is just a `source` config right?

Reply:
Yup this was copied over we should delete